inc/com/centreon/engine/command_manager.hh

```diff
@@ -31,6 +31,8 @@
 CCE_BEGIN()
 class command_manager {
```

Comment:
Could you explain in Doxygen format the new introduced variables.

What _queue is for, what _has_data does?